src/Waargonaut/Generic.hs

```diff
@@ -53,16 +54,20 @@ module Waargonaut.Generic
   ) where
 
 import           Generics.SOP
+import           Generics.SOP.Record (IsRecord)
 
-import           Control.Lens                  (findOf, folded, isn't, _Left)
+import           Control.Lens                  (findOf, folded, isn't, _Left, _Empty, (#))
 import           Control.Monad                 ((>=>))
 import           Control.Monad.Except          (lift, throwError)
 import           Control.Monad.Reader          (runReaderT)
 import           Control.Monad.State           (modify)
 
+import Data.Function ((&))
```

Comment:
This line and the foldable import below don't follow your import indentation conventions